### PR TITLE
CSS improvements

### PR DESCRIPTION
**Pull request by 1ctinus**
fixed variable and h1 indentation. Why? honestly, im bored and want to start small on some repositories that arent mine

### CODE REVIEW

**Comment by mugman174** [comment]
imma test this

**Review by thatguysawesome — APPROVED** [review]
no idea why this pull request exists but ok

**Comment by 1ctinus** [comment]
@

> no idea why this pull request exists but ok

im bored.

**Comment by caughtquick** [comment]
Looks fine to me, merging now :+1:

File: source/styles/base.css [hunk 1]
```diff
@@ -116,3 +116,6 @@ body {
 	background: #58a5f0;
 	border: 2px solid #0277bd;
 }
+h1 {
```
Comment (mugman174) [inline source/styles/base.css:119]: Before: before
After: after

**Comment by mugman174** [comment]
@caughtquick let me test next time

File: source/styles/base.css (hunk 1 above)
Comment (caughtquick) [reply source/styles/base.css:119]: No it looks better on top, plus its already merged so like :man_shrugging:

**Pull request merged**